packages/backend/database/src/drizzle/schema.ts

```diff
@@ -0,0 +1,454 @@
+  },
+  (table) => ({
+    resourceViewsUserIdx: index("resource_views_user_idx").on(table.userId),
+    resourceViewsResourceIdx: index("resource_views_resource_idx").on(
+      table.resourceId
+    ),
+    resourceViewsSessionIdx: index("resource_views_session_idx").on(
+      table.sessionId
+    ),
+  })
+);
+
+export const resourceSaved = pgTable(
+  "resource_saved",
+  {
+    id: integer("id").primaryKey(),
+    resourceId: integer("resource_id").notNull(),
+    userId: integer("user_id"),
+    savedAt: timestamp("saved_at").defaultNow(),
+  },
+  (table) => ({
+    resourceSavedResourceIdx: index("resource_saved_resource_idx").on(
+      table.resourceId
+    ),
+    resourceSavedUserIdx: index("resource_saved_user_idx").on(table.userId),
+  })
+);
+
+export const company = pgTable(
+  "company",
+  {
+    id: integer("id").primaryKey().notNull(),
+    name: varchar("name", { length: 255 }).unique(),
+    description: text("description"),
+    website: varchar("website", { length: 255 }).unique(),
+    size: varchar("size", { length: 10 }),
+    location: varchar("location", { length: 255 }),
+    industry: varchar("industry", { length: 255 }),
+    isVisible: boolean("is_visible").default(false),
+    instagramUrl: varchar("intagram_url", { length: 255 }),
+    facebookUrl: varchar("facebook_url", { length: 255 }),
+    xUrl: varchar("x_url", { length: 255 }),
+    linkedinUrl: varchar("linkedin_url", { length: 255 }),
+    createdAt: timestamp("created_at").notNull().defaultNow(),
+    updatedAt: timestamp("updated_at").notNull().defaultNow(),
+  },
+  (table) => ({
+    companyNameIdx: uniqueIndex("company_name_idx").on(table.name),
+    companyWebsiteIdx: uniqueIndex("company_website_idx").on(table.website),
+    companyIndustryIdx: index("company_industry_idx").on(table.industry),
+    companyIsVisibleIdx: index("company_is_visible_idx").on(table.isVisible),
+  })
+);
+
+export const companyFeedback = pgTable(
+  "company_feedback",
+  {
+    id: integer("id").primaryKey().notNull(),
+    companyId: integer("company_id"),
+    userId: integer("user_id"),
+    description: text("description"),
+    approved: boolean("approved").default(false),
+    createdAt: timestamp("created_at").notNull().defaultNow(),
+    updatedAt: timestamp("updated_at").notNull().defaultNow(),
+  },
+  (table) => ({
+    companyFeedbackCompanyIdx: index("company_feedback_company_idx").on(
+      table.companyId
+    ),
+    companyFeedbackUserIdx: index("company_feedback_user_idx").on(table.userId),
+    companyFeedbackApprovedIdx: index("company_feedback_approved_idx").on(
+      table.approved
+    ),
+  })
+);
+
+export const tags = pgTable(
+  "tags",
+  {
+    id: integer("id").primaryKey(),
+    name: varchar("name", { length: 255 }).notNull().unique(),
+  },
+  (table) => ({
+    tagsNameIdx: uniqueIndex("tags_name_idx").on(table.name),
+  })
+);
+
+// Relation tables
+export const userViewsJob = pgTable(

```

Comment:
Does this map the relations ? if so, wouldn't it be simpler to add the references in the main tables instead ?